### DISCUSSION TITLE
Does MRT Core support binary version info too?

### DISCUSSION THREAD

**Discussion by MarkIngramUK** [microsoft/WindowsAppSDK#975] [Q&A] [answered]
As MRT Core replaces the traditional `resources.rc` and `resource.h` combo, does it have a solution for binary version info? i.e. the version info that would normally go in `resources.rc` similar to:

```
VS_VERSION_INFO VERSIONINFO

 FILEVERSION 1,2,3,4

 FILEFLAGSMASK 0x3fL
#ifdef _DEBUG
 FILEFLAGS 0x1L
#else
 FILEFLAGS 0x0L
#endif
 FILEOS 0x40004L
 FILETYPE 0x1L
 FILESUBTYPE 0x0L
BEGIN
    BLOCK "StringFileInfo"
    BEGIN
        BLOCK "080904b0"
        BEGIN
            VALUE "CompanyName", "Company"
            VALUE "LegalCopyright", "Copyright"
            VALUE "ProductName", "App Name"
            VALUE "FileDescription", "App Description"
            VALUE "OriginalFilename", "App.exe"
            VALUE "ProductVersion", "1.2.3.4"
        END
    END
    BLOCK "VarFileInfo"
    BEGIN
        VALUE "Translation", 0x809, 1200
    END
END
```

**Comment by axelandrejs** [comment, marked as answer]
There is currently no support for this. The problem is mostly on the other side of the equation, these values are read by the OS, and the OS does not understand MRT resources in this context.

**Reply by MarkIngramUK** [reply]
OK, that's understandable. It'd be nice if we had a `resversion` (similar to `resw` / `resfiles`) that could be embedded into a binary, in the same way as `resource.rc` would be. Just using a more "modern" format, then the OS can continue to read the version info in the same way.
